graphql/task_logs_resolver.go

```diff
@@ -195,6 +202,9 @@ func (r *taskLogsResolver) TaskLogs(ctx context.Context, obj *TaskLogs) ([]*apim
 		taskLogReader, err := apimodels.GetBuildloggerLogs(ctx, opts)
 
 		if err != nil {
+			if task.Status == evergreen.TaskUndispatched && strings.Contains(err.Error(), notFoundMessage) {
```

Comment:
We should use this. https://github.com/evergreen-ci/evergreen/blob/c082c88e7620c00df81d5e4d528fbdd144d27312/globals.go#L336